advocacy_docs/pg_extensions/index.mdx

```diff
@@ -16,19 +16,16 @@ Categories of extensions:
 -   [EDB extensions](#edb-extensions)
 -   [EDB supported open-source extensions](#edb-supported-open-source-extensions)
 
-!!! Note
-    For BigAnimal:
-
-    -   PostgreSQL compatible corresponds to EDB Postgres Extended Server.
-    -   Oracle compatible corresponds to EDB Postgres Advanced Server.
+!!! Note Access requirements for BigAnimal
+    See [Managing extensions](/biganimal/latest/using_cluster/03_modifying_your_cluster/extensions/) in the BigAnimal documentation.
 
 
 
 ## Open source extensions
```

Comment:
I think we should remove the "Requires superuser" column from this table. It's a bit weird to include now that it's meaningless for BigAnimal. /cc @kelpoole

Reply:
Is it really meaningless for BigAnimal? Doesn't it indicate what extensions require superuser for BYOA users?

Reply:
At that point what will the list of extensions available on BA look like? will it be a handful? or the 40 or so that the table says are supported on BA and don't require superuser?